src/main/java/org/mdbenefits/app/utils/SubmissionUtilities.java

```diff
@@ -89,7 +90,7 @@ public static String formatMoney(Double value) {
     }
 
     public static boolean isNolaParish(Submission submission) {
-        return Parish.ORLEANS.name().equals(submission.getInputData().get("parish"));
```

Comment:
Can't we get rid of the calls to this method easily?

Reply:
I'll see if it opens up a rabbit hole. If not, I'll take it and calls to it out.